app/assets/stylesheets/ncr/_fields.scss

```diff
@@ -0,0 +1,10 @@
+#description-wrapper {
```

Comment:
Apparently the Front End Guild strongly advises against styling IDs (versus classes) because they're much harder to override.
(I have a sneaking suspicion that I've done this elsewhere in the redesign, so I should go find & fix it)